spec/system/topic_bulk_select_spec.rb

```diff
@@ -37,5 +38,54 @@
       topic_list_header.click_bulk_topics_confirm
       expect(topic_list).to have_closed_status(topics.first)
     end
+
+    it "closes topics normally" do
+      # Watch the topic as a user
+      sign_in(user)
+      visit("/latest")
+      topic = topics.third
+      visit("/t/#{topic.slug}/#{topic.id}")
+      topic_page.watch_topic
+      sleep(2) # Needs to wait to trigger the read state of the topic.
```

Comment:
I suspect this will be flaky. An alternative might be to trigger a `topic timings` update, i.e. same as what the frontend does under the hood to mark a topic/post as read: 

```
      PostTiming.create!(
        topic_id: post.topic_id,
        post_number: post.post_number,
        user_id: user.id,
        msecs: 100,
      )
```